ci/infra/vmware/lb-instance.tf

```diff
@@ -148,8 +148,10 @@ resource "vsphere_virtual_machine" "lb" {
   hardware_version = var.vsphere_hardware_version
 
   disk {
-    label = "disk0"
-    size  = var.lb_disk_size
+    label            = "disk0"
+    size             = var.lb_disk_size
+    eagerly_scrub    = data.vsphere_virtual_machine.template.disks.0.eagerly_scrub
+    thin_provisioned = data.vsphere_virtual_machine.template.disks.0.thin_provisioned
```

Comment:
Yes, thin provision is a good improvement.

Reply:
FI, this was already using thin provisioning since it's the default in terraform